pkg/component/component.go

```diff
@@ -937,6 +946,28 @@ func List(client *occlient.Client, applicationName string, localConfigInfo *conf
 	return compoList, nil
 }
 
+// List lists all components in active application
+func List(client *occlient.Client, applicationName string, localConfigInfo *config.LocalConfigInfo) (ComponentList, error) {
+	var applicationSelector string
+	if applicationName != "" {
+		applicationSelector = fmt.Sprintf("%s=%s", applabels.ApplicationLabel, applicationName)
```

Comment:
Why not use  `applicationSelector = applabels.GetSelector(applicationName)` as is done in other places?

Reply:
done